specification/purview/data-plane/Azure.Analytics.Purview.SelfServicePolicies/preview/2022-12-01-preview/purviewSelfServicePolicy.json

```diff
@@ -0,0 +1,348 @@
+        "purviewRoleName": {
+          "type": "string",
+          "description": "This is field will indicate the role of a self service policy."
+        },
+        "principal": {
+          "type": "string",
+          "description": "Azure Active Directory Id for users"
+        },
+        "principalGroup": {
+          "type": "string",
+          "description": "Azure Active Directory Id for groups"
+        },
+        "resourceAzurePath": {
+          "type": "string",
+          "description": "The resource path of the azure resource"
+        }
+      }
+    },
+    "ErrorModel": {
+      "description": "The error model for policy",
+      "required": [
+        "code",
+        "message"
+      ],
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "The error code",
+          "minLength": 1,
+          "type": "string"
+        },
+        "message": {
+          "description": "The error message",
+          "minLength": 1,
+          "type": "string"
+        },
+        "details": {
+          "description": "The error details",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorModel"
+          }
+        }
+      }
+    },
+    "ErrorResponseModel": {
+      "description": "The error response model for policy",
+      "required": [
+        "error"
+      ],
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorModel"
+        }
+      }
+    },
+    "SelfServicePolicy": {
+      "description": "The Self-service policy model",
+      "required": [
+        "decisionRule",
+        "workflowRunId"
+      ],
+      "type": "object",
+      "properties": {
+        "workflowRunId": {
+          "type": "string",
+          "description": "The workflowRunId of the workflow which created the policy"
+        },
+        "requestor": {
+          "type": "string",
+          "description": "The requestor of the self-service policy"
+        },
+        "expiryDate": {
+          "format": "date-time",
+          "type": "string",
+          "description": "Expiry Date of the policy"
+        },
+        "decisionRule": {
+          "$ref": "#/definitions/SelfServicePolicyDecisionRule"
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData"
+        }
+      }
+    },
+    "SelfServicePoliciesList": {
+      "description": "List of Self-service policies",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of policies",
+          "items": {
+            "$ref": "#/definitions/SelfServicePolicy"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Pagination link"
+        }
+      }
+    },
+    "SystemData": {
+      "type": "object",
+      "description": "The system data",
+      "readOnly": true,
```

Comment:
`readOnly: true` has no significance for schemas that are only used in responses, so you should remove this.